libs/db_lib/mysqli.php

```diff
@@ -24,7 +24,7 @@ function connect($db_host, $db_username, $db_password, $db_name = NULL, $use_nam
 
         if ($this->link_id){
             if (!empty($use_names)) $this->query("SET NAMES '$use_names'");
-        } else die($lang_global['err_sql_conn_db']);
+        } else die($lang_global['err_sql_conn_db'] . " DB: {$db_name}");
```

Comment:
Added this to make it more clear which database is failing, since the default error language does not make it obvious, and there's several potential culprits